### DISCUSSION TITLE
FJC IDB could use an update

### DISCUSSION THREAD

**Discussion by kfinity** [freelawproject/courtlistener#3668] [General]
So I loaded the CL bulk data from December 2023, and the most recent date_filed in recap_fjcintegrateddatabase is 2022-03-01. Loading the latest IDB from the FJC site gets me up through 2023-09-30. 
Do you all want any help loading fresh data, either manually or in some sort of automated fashion?

**Comment by mlissner** [comment]
Yeah, we really only update it when somebody says it's out of date and asks for an update. We could add this to our backlog, if that'd be helpful. I don't think there's a good way for a non-staff member to help with this, unfortunately.

**Reply by kfinity** [reply]
No worries! I don't really need it updated for my current project, so please don't rush on my account. Just curious how it was working. Thanks for the quick response.
